package.json

```diff
@@ -1,78 +1,86 @@
 {
-  "name": "finance-endpoints",
-  "version": "0.0.0",
+  "name": "dcCampaignFinanceWatch",
+  "private": true,
+  "version": "0.0.1",
   "description": "",
-  "main": "server.js",
-  "engines": {
-    "node": "0.10.20"
-  },
+  "main": "",
   "scripts": {
-    "start": "node app.js",
-    "dev": "gulp serve --env=local",
-    "test": "mocha --compilers js:babel-core/register --recursive --reporter spec",
-    "test:watch": "npm test -- --watch",
-    "cover": "babel-node ./node_modules/istanbul/lib/cli cover -- _mocha --recursive --reporter spec"
-  },
-  "author": "",
-  "license": "BSD-2-Clause",
-  "dependencies": {
-    "bluebird": "^2.10.2",
-    "bunyan": "^1.2.3",
-    "d3": "^3.5.6",
-    "fast-csv": "^1.0.0",
-    "fbjs": "^0.2.1",
-    "gm": "^1.21.1",
-    "jquery": "^2.2.1",
-    "lodash": "^4.6.1",
-    "markdownify": "^0.1.0",
-    "moment": "^2.10.6",
-    "moment-range": "^2.0.3",
-    "mongodb": "^2.1.8",
-    "mongoose": "4.4.5",
-    "react": "^0.14.7",
-    "react-bootstrap": "^0.28.4",
-    "react-day-picker": "^1.3.1",
-    "react-dom": "^0.14.2",
-    "react-router": "^2.0.1",
-    "redis": "^2.4.2",
-    "redis-jsonify": "^1.0.0",
-    "rest": "^1.3.1",
-    "restify": "~2.8.3",
-    "restler": "^3.2.2",
-    "rx": "^4.0.7",
-    "rx-react": "^0.3.0",
-    "svg-to-png": "^2.0.1",
-    "svg2png": "^2.0.0",
-    "underscore": "^1.8.3"
+    "clean": "rimraf dist/*",
+    "copy": "copyfiles -f ./src/index.html ./src/favicon.ico ./dist",
+    "dist": "npm run copy & webpack --env=dist",
+    "lint": "eslint ./src",
+    "posttest": "npm run lint",
+    "release:major": "npm version major && npm publish && git push --follow-tags",
+    "release:minor": "npm version minor && npm publish && git push --follow-tags",
+    "release:patch": "npm version patch && npm publish && git push --follow-tags",
+    "serve": "node server.js --env=dev",
```

Comment:
We need to modify one of these serve commands to support pointing to the production api.